components/background_hang_monitor/background_hang_monitor.rs

```diff
@@ -51,7 +51,7 @@ impl BackgroundHangMonitorRegister for HangMonitorRegister {
     ) -> Box<BackgroundHangMonitor> {
         let bhm_chan = BackgroundHangMonitorChan::new(self.sender.clone(), component_id);
 
-        #[cfg(target_os = "windows")]
+        #[cfg(all(target_os = "windows", not(target_arch = "aarch64")))]
```

Comment:
Better? to list the architectures that are supported than the one that isn't?

Reply:
Also, does the catch-all `DummySampler` not need to be enabled for windows aarch64?

Reply:
It's already enabled by `#[cfg(any(target_os = "android", target_arch = "arm", target_arch = "aarch64"))]`.